File: 0965-unique-email-addresses/0965-unique-email-addresses.cs
public class Solution {
    public int NumUniqueEmails(string[] emails) =>
        emails.Select(email => email.Substring(0, System.Text.RegularExpressions.Regex.Match(email, "[+@]").Index).Replace(".", "") + email.Substring(email.IndexOf('@'))).Distinct().Count();
}

File: 0965-unique-email-addresses/0965-unique-email-addresses.js
/**
 * @param {string[]} emails
 * @return {number}
 */
var numUniqueEmails = function(emails) {
  return new Set(emails.map(email => email.substr(0, email.search(/[+@]/)).replaceAll('.', '') + email.substr(email.search('@')))).size
};